tests/unit/components/playback/controls/AddCustomer.spec.js

```diff
@@ -361,7 +364,6 @@ describe("AddCustomer.clicked_button", () => {
       "",
       "",
       "color: rgb(255, 255, 255); width: 250px; left: 250px;",
-      1,
     ],
   ])(
     "Given an UUID(%s) , API Key(%s), Nickname(%s) for 'Add Customer' as input, When the input contains based on valid the critera or failure %s %s %s, Then display of Label 'Save ID' is visible %s, click on Cancel, emitted event (value 0) and click on Save emitted event (value %s)",
```

Comment:
it looks like this test name needs to be updated to reflect the changes you made (for example `value 0` is not part of the test anymore).  Please apply this comment to the tests for all 4 components

Reply:
Agreed .. correction done on title of the test.